docs/architecture/cloud-native/logging-with-elastic-stack.md

```diff
@@ -18,17 +19,17 @@ Elastic Stack provides centralized logging in a low-cost, scalable, cloud-friend
 
 The first component is [Logstash](https://www.elastic.co/products/logstash). This tool is used to gather log information from a large variety of different sources. For instance, Logstash can read logs from disk and also receive messages from logging libraries like [Serilog](https://serilog.net/). Logstash can do some basic filtering and expansion on the logs as they arrive. For instance, if your logs contain IP addresses then Logstash may be configured to do a geographical lookup and obtain a country or even city of origin for that message.
 
-Serilog is a logging library for .NET languages, which allows for parameterized logging. Instead of generating a textual log message that embeds fields, parameters are kept separate. This allows for more intelligent filtering and searching. A sample Serilog configuration for writing to Logstash appears in Figure 7-2.
+Serilog is a logging library for .NET languages, which allows for parameterized logging. Instead of generating a textual log message that embeds fields, parameters are kept separate. This allows for more intelligent filtering and searching. A sample Serilog configuration for writing to Logstash appears in Figure 7-5.
 
 ```csharp
 var log = new LoggerConfiguration()
          .WriteTo.Http("http://localhost:8080")
          .CreateLogger();
 ```
 
-**Figure 7-2** Serilog config for writing log information directly to logstash over HTTP
+**Figure 7-5** Serilog config for writing log information directly to logstash over HTTP
```

Comment:
in the previous article, you used a period after the number. Here you don't use anything and then on 7.6, you use a dash. Maybe standardize?

Reply:
Good catch. Should be period throughout.